homeassistant/components/dpi4relay/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "dpi4relay",
+  "name": "DockerPi 4 Channel Relay expansion board",
```

Comment:
The name is very elaborate, could you shorten it a bit?
How about "DockerPi 4 Channel Relay"?